backend/tracim_backend/lib/core/content.py

```diff
@@ -1045,7 +1046,9 @@ def get_jpg_preview_path(
                 height=height,
                 file_ext=file_extension,
             )
-        except UnsupportedMimeType as exc:
+        except (PreviewDimNotAllowed, PageOfPreviewNotFound) as exc:
+            raise exc
+        except Exception as exc:
```

Comment:
In that case it is preferable to explain in message what we caught an unkown error

Reply:
done